run.sh

```diff
@@ -1,12 +1,16 @@
+#!/bin/bash
+
+set -ex
+
 version=1.5
 # version=master
 iteration=0
 
-sudo luarocks remove apisix --local
+sudo luarocks remove apisix --local || true
```

Comment:
this is wrong. we need to remove the apisix in folder `/tmp/apisix/usr/local/apisix/deps`

it should be a bug for a long time

Reply:
I think we should use `luarocks purge --tree=/tmp/apisix/usr/local/apisix/deps` here

Reply:
or we can clean the folder `/tmp/apisix/usr/local/apisix/deps`, this is a simpler way

Reply:
fixed.